easybuild/easyblocks/generic/rpackage.py

```diff
@@ -152,23 +149,6 @@ def make_cmdline_cmd(self, prefix=None):
         self.log.debug("make_cmdline_cmd returns %s" % cmd)
         return cmd, None
 
-    def extract_step(self):
-        """Source should not be extracted."""
-
-        if self.cfg['unpack_sources']:
-            super(RPackage, self).extract_step()
-        elif len(self.src) > 1:
-            raise EasyBuildError("Don't know how to handle R packages with multiple sources.'")
-        else:
-            try:
-                shutil.copy2(self.src[0]['path'], self.builddir)
-            except OSError as err:
-                raise EasyBuildError("Failed to copy source to build dir: %s", err)
-            self.ext_src = self.src[0]['name']
-
-            # set final path since it can't be determined from unpacked sources (used for guessing start_dir)
-            self.src[0]['finalpath'] = self.builddir
```

Comment:
How is it OK to drop this custom `extract_step` implementation?

Reply:
Yes, In https://github.com/easybuilders/easybuild-easyblocks/pull/1938#issuecomment-582362995 I suggested to always extract, see also PR description. Hence there is no need to do the "maybe extract or copy"-dance

Reply:
Thanks for the clarification!